README.md

```diff
@@ -41,19 +30,35 @@ These environment variables can be useful:
 ###  :running: Docker + Lambda
 If you want to test the lambda with high fidelity (more [here](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html)) you can do the following steps (adjust, if needed):
 ```
+# Start a local Elasticsearch cluster. Notice the network it uses: "es-net".
+docker-compose up 
+
 # Build the image
 docker build -t <name-of-your-image> .
 
 # Run the lambda the container
-docker run --rm -it -p 9000:8080 <name-of-your-image>
+ docker run --rm -it --network="es-net" -p 9000:8080 -e es_host="elasticsearch" <name-of-your-image>
 
-# Invoke the lambda
-curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{ "Record": <...> }'
-
-For instance,
-curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{"Records":[{"messageId":"059f36b4-87a3-44ab-83d2-661975830a7d","body":{"_id":"abc","firstName":"John","lastName":"Doe","hashedEmail":"0bc83cb571cd1c50ba6f3e8a78ef1346","email":"test@yahoo.ca"}},{"messageId":"2e1424d4-f796-459a-8184-9c92662be6da","body":{"_id":"def","firstName":"Jane","lastName":"River","isPublic":"True","interests":["Volley Ball","Natation"],"virtualStudies":[{"_id":"1","name":"Virtual Study 1"},{"_id":"12","name":"Virtual Study 2"}]}}]}'
+# Invoke the lambda. For instance,
+curl --location --request POST 'http://127.0.0.0:9000/2015-03-31/functions/function/invocations' \
```

Comment:
We should be using `localhost` instead of `127.0.0.0` or `127.0.0.1`.

Would also be good to note the expected output from a successful curl, which is `null`

```suggestion
curl --location --request POST 'http://localhost:9000/2015-03-31/functions/function/invocations' \
```